app/app.go

```diff
@@ -304,12 +380,15 @@ func NewTgradeApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	)
 
 	govRouter.AddRoute(twasm.RouterKey, twasmkeeper.NewProposalHandler(app.twasmKeeper))
-
-	var ibcHandler porttypes.IBCModule = wasm.NewIBCHandler(app.twasmKeeper, app.ibcKeeper.ChannelKeeper)
-	ibcRouter.AddRoute(twasm.ModuleName, ibcHandler)
+	ibcRouter.AddRoute(wasm.ModuleName, wasm.NewIBCHandler(app.twasmKeeper, app.ibcKeeper.ChannelKeeper))
 	app.ibcKeeper.SetRouter(ibcRouter)
 
-	app.poeKeeper = poekeeper.NewKeeper(appCodec, keys[poe.StoreKey], app.getSubspace(poe.ModuleName), app.twasmKeeper)
+	app.poeKeeper = poekeeper.NewKeeper(
```

Comment:
BTW, I noticed in Osmsosis, all these keepers are public members of app.
It does make testing simpler.

Why not make them public? It's not like any modules have access to this, but most of the test code does (when it instantiates an App). It would make some test code simpler